examples/tv-casting-app/tv-casting-common/core/CastingPlayer.h

```diff
@@ -197,6 +202,17 @@ class CastingPlayer : public std::enable_shared_from_this<CastingPlayer>
     std::vector<memory::Strong<Endpoint>> mEndpoints;
     ConnectionState mConnectionState = CASTING_PLAYER_NOT_CONNECTED;
     CastingPlayerAttributes mAttributes;
+    IdentificationDeclarationOptions mIdOptions;
+    /**
+     * Flag to indicate whether or not the Commissionee (tv-casting-app) is enabled for the Commissioner-Generated Passcode flow.
+     */
+    bool mUdcCommissionerPasscodeEnabled = false;
+    /**
+     * Flag to indicate whether or not the Commissionee has obtained the Commissioner Passcode from the user and is therefore ready
+     * for commissioning.
+     */
+    bool mUdcCommissionerPasscodeReady                               = false;
+    CommissionerDeclarationHandler * mCommissionerDeclarationHandler = new CommissionerDeclarationHandler();
```

Comment:
q: Are we supposed to have one CommissionerDeclarationHandler per CastingPlayer object (which as per current change in this PR), or is one CommissionerDeclarationHandler common across all commissioning attempts?

Reply:
As discusses, we will have one CommissionerDeclarationHandler across all commissioning attempts and CastingPlayer objects. The API's consumer will be responsible for tracking which CastingPlayer they are connecting to and handle the callback appropriately.